backend/core/endpoints/channels.py

```diff
@@ -64,11 +64,9 @@ def create_channel(
         if server is None:
             raise HTTPException(status_code=404, detail="Server not found")
         channel = schemas.ChannelCreate(
```

Comment:
Probably should discuss if we want to change this everywhere or leave it the old way